api/src/pcapi/core/booking_providers/cds/CineDigitalService.py

```diff
@@ -30,3 +32,13 @@ def get_payment_type(self) -> PaymentTypeCDS:
             f"Pass Culture payment type not found in Cine Digital Service API for cinemaId={self.cinemaid}"
             f" & url={self.apiUrl}"
         )
+
+    def get_tariff(self) -> TariffCDS:
+        tariffs = get_tariffs(self.cinemaid, self.apiUrl, self.token)
+        for tariff in tariffs:
+            if tariff.label == "Pass Culture 5€":
+                return tariff
```

Comment:
Je pense qu'on peux déplacer ça dans la fonction du connector comme tu la fais pour les modes de paiement, on aura pas besoin de la liste de tous les tarifs.

Reply:
J'ai changé dans payment pour coller à ta façon de faire dans get_shows
Ca nous faciliera le refactoring car les fonctions sont construites de manière identique, on pourra retirer pas mal de répétition

Reply:
On purra faire un refactoring en passant 
la route et le type de données à filtrer 
exemple:

`def get_shows(cinema_id: str, url: str, token: str) -> list[ShowCDS]:`
on pourra remplacer par une fonction générique :

```suggestion
    def get_resource_from_cds_api(cinema_id: str, url: str, token: str, resource: str, resource_type : dataclass, mocked_resource : list) -> list[ShowCDS]:
         api_url = f"https://{cinema_id}.{url}{resource}?api_token={token}"

    try:
        if not settings.IS_DEV:
            api_response = requests.get(api_url)
        else:
            api_response = mocked_resource()
    except Exception:
        raise cds_exceptions.CineDigitalServiceAPIException(
            f"Error connecting CDS for cinemaId={cinema_id} & url={url}"
        )

    if api_response.status_code != 200:
        raise cds_exceptions.CineDigitalServiceAPIException(
            f"Error getting Cine Digital Service API DATA for cinemaId={cinema_id} & url={url}"
        )

    json_response = api_response.json()
    ressource_list = parse_obj_as(list[resource_type], json_response)

    return ressource_list 
```